### DISCUSSION TITLE
Wrapper for stripe prices

### DISCUSSION THREAD

**Discussion by leonwetterich** [supabase/supabase#26643] [Contribute to Supabase]
Are you planning on implementing the pricing table from stripe to the wrapper? It would be really useful since you only get a price id from the products table.

**Comment by nielsbrakel** [comment]
This would be great!

**Reply by nielsbrakel** [reply]
I created a PR for this https://github.com/supabase/wrappers/pull/84

**Reply by nielsbrakel** [reply]
The change was just merged
